### PR TITLE
Add Draco IO plugin

### PR DESCRIPTION
**Pull request by dgirardeau**
Early version

### CODE REVIEW

File: plugins/core/Standard/CMakeLists.txt [hunk 1]
```diff
@@ -20,6 +20,7 @@ set( submod_plugins
 		${CMAKE_CURRENT_SOURCE_DIR}/qMasonry
 		${CMAKE_CURRENT_SOURCE_DIR}/qMPlane
 		${CMAKE_CURRENT_SOURCE_DIR}/qJSonRPCPlugin
+		${CMAKE_CURRENT_SOURCE_DIR}/qUltrabeam
```
Comment (tmontaigu) [inline plugins/core/Standard/CMakeLists.txt:23]: Is qUltrabeam a part of the qDracoIO plugin ? 🤔

File: plugins/core/IO/qDracoIO/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+option( PLUGIN_IO_QDRACO "Install qDracoIO plugin to read & write DRC files" OFF )
+
+if( PLUGIN_IO_QDRACO )
+	project( QDRACO_IO_PLUGIN )
+
+	set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
```
Comment (tmontaigu) [inline plugins/core/IO/qDracoIO/CMakeLists.txt:6]: It seems the files contained in this directory have not been added to the commit
And `list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)` might be better to not override the `CMAKE_MODULE_PATH`

**Comment by tmontaigu** [comment]
I see there is a [conda-forge package](https://anaconda.org/conda-forge/draco) for draco so we should be able to add the plugin to the CI of Windows and macOs

File: plugins/core/Standard/CMakeLists.txt (hunk 1 above)
Comment (dgirardeau) [reply plugins/core/Standard/CMakeLists.txt]: Good point, this one shouldn't even be here 👍

File: plugins/core/IO/qDracoIO/CMakeLists.txt (hunk 2 above)
Comment (dgirardeau) [reply plugins/core/IO/qDracoIO/CMakeLists.txt]: Ah, that was a (failing) attempt to help CMake find the module...

**Comment by dgirardeau** [comment]
> I see there is a [conda-forge package](https://anaconda.org/conda-forge/draco) for draco so we should be able to add the plugin to the CI of Windows and macOs

I don't know!

**Pull request merged**